velox/functions/prestosql/tests/JsonFunctionsTest.cpp

```diff
@@ -67,6 +74,171 @@ TEST_F(JsonFunctionsTest, jsonArrayLength) {
   EXPECT_EQ(json_array_length(R"({"k1":[0,1,2], "k2":"v1"})"), std::nullopt);
 }
 
+TEST_F(JsonFunctionsTest, jsonArrayContainsBool) {
+  EXPECT_EQ(json_array_contains<bool>(R"([])", 0), false);
+  EXPECT_EQ(json_array_contains<bool>(R"([1, 2, 3])", 1), false);
+  EXPECT_EQ(json_array_contains<bool>(R"([1.2, 2.3, 3.4])", 2.3), false);
+  EXPECT_EQ(
+      json_array_contains<bool>(
```

Comment:
For test cases of json arrays whose element types are not bool, let's use `true` or `false` as the second argument for the json_array_contains call to avoid possible confusion.